interface/usergroup/usergroup_admin_add.php

```diff
@@ -645,6 +654,51 @@ function authorized_clicked() {
 
 });
 </script>
+
+<!-- Password Strength Meter JavaScript -->
+<script>
+function checkPasswordStrength() {
+    var number = /(\p{N})/u;
+    var alphabets = /(\p{L})/u;
+    var special_characters = /([~,!,@,#,$,%,^,&,*,-,_,+,=,?,>,<])/;
```

Comment:
would it simply make more sense and broaden special characters (emojis etc) to have it just be:
special character = Not a Number or a Letter

Also, does this capture numbers/letters in other languages? (just curious)

Reply:
Made the above change and yet it does capture numbers and letters in other languages.